gocourse05/server/log_sevice.go

```diff
@@ -0,0 +1,31 @@
+package server
+
+import (
+	"os"
+	"time"
+)
+
+type LogService struct {
+	strangeApi string // idk just to be...
```

Comment:
```suggestion
	strangeAPI string // idk just to be...
```

https://go.dev/wiki/CodeReviewComments#initialisms